README.md

```diff
@@ -334,13 +342,22 @@ As well as some other IBM internal compilers, and LLVM projects.
 
 * Compiler engineering
 
+## [NoRedInk](https://www.noredink.com/jobs)
+
+* Elm
+* [Roc](https://www.roc-lang.org/)
+
```

Comment:
It's not clear from their site if NoRedInk actually helps build Elm (or Roc?) 

Without a job posting I'd prefer not to include them here.

Reply:
Evan Czaplicki, the creator of Elm, currently works at NoRedInk. That said, the [compiler repo hasn't had much activity lately](https://github.com/elm/compiler/commits/master), so maybe it makes sense to remove that.

Richard Feldman who is the Head of Tech at NoRedInk recently gave a talk on Roc: https://twitter.com/rtfeldman/status/1441558046149337088 .

That said, they're relatively small, so it's quite possible they don't have openings to work on this anytime in the near future. So I'm going to remove this.